ixmp/core.py

```diff
@@ -925,17 +925,7 @@ def par(self, name, filters=None, **kwargs):
         filters : dict
             index names mapped list of index set elements
         """
-        result = self._backend('item_get_elements', 'par', name, filters)
-
-        # FIXME message_ix requires 'year' columns to be returned as integers
-        #       This code should be in a message_ix override of this method.
-        dtypes = {}
-        for idx_set, col_name in zip(self.idx_sets(name),
-                                     self.idx_names(name)):
-            if idx_set == 'year':
-                dtypes[col_name] = int
-
-        return result.astype(dtypes) if len(dtypes) else result
+        return self._backend('item_get_elements', 'par', name, filters)
```

Comment:
nit: Should `kwargs` be passed to backend's `item_get_elements` method?

Reply:
I think these `kwargs` should be removed entirely; they exist only to be forgiving of sloppy user code.